addons/website_event/static/src/scss/event_templates_page.scss

```diff
@@ -9,6 +9,13 @@
 
     // Ensure menu labels are not split
     #o_wevent_event_submenu {
+        ul {
+            justify-content: center;
```

Comment:
Can't you use bootstrap classes like d-flex, ... ? I know there are people doing css and people using bootstrap; but to be coherent with event coding style we use bootstrap classes currently :) .

Reply:
Yes, indeed, we can easily replace these with bootstrap classes :)